### PR TITLE
Issue 1240 tos update

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Show the BrowserID and RP TOS/PP at the correct time with the correct text.

To Test:

```
* BID TOS/PP is only displayed to new users.
* RP TOS/PP is only displayed to new users, users who are visiting a new site, or users who have just authenticated to BrowserID.
```

### CODE REVIEW

File: resources/static/dialog/css/popup.css [hunk 1]
```diff
@@ -284,24 +284,24 @@ section > .contents {
 #favicon img {
     display: block;
     margin: 0 auto;
-    max-height: 128px;
-    max-width: 128px;
+    max-height: 100px;
+    max-width: 100px;
```
Comment (lloyd) [inline resources/static/dialog/css/popup.css]: this is an important change.  100x100 is our new siteLogo maxsize...

File: resources/static/test/cases/controllers/dialog.js [hunk 2]
```diff
@@ -151,6 +151,7 @@
         catch(e) {
           // do nothing, an exception will be thrown because no modules are
           // registered for the any services.
+          console.log(e);
```
Comment (lloyd) [inline resources/static/test/cases/controllers/dialog.js]: intentional console.log() ?

Comment (shane-tomlinson) [inline resources/static/test/cases/controllers/dialog.js]: @lloyd - good eye - this console.log message is intentional since it is in the tests.

Comment (shane-tomlinson) [inline resources/static/test/cases/controllers/dialog.js]: @lloyd I removed this console.log message in 0a4c08e3b9747463593d6821138be48987a33abe

**Comment by shane-tomlinson** [comment]
Link to issue #1240

**Comment by lloyd** [comment]
r+ - tests clean, walkthrough with skinny looks good.  expect some instability but we'll knock it down as it comes up.

**Pull request merged**